www/js/overlay.js

```diff
@@ -119,27 +127,26 @@ const decreaseOpacity = () => {
 };
 
 const separator = h('div.separator');
+const separatorY = () => h('div.separator-y').cloneNode(true);
 
 const layoutButtonFactory = layoutName =>
   h(
-    'div.input-wrap',
-    h(
-      `div.layout-btn.${layoutName}`,
-      h('div.layout-bar.layout-bar-1'),
-      h('div.layout-bar.layout-bar-2'),
-      h('div.layout-bar.layout-bar-3'),
-      h('div.layout-bar.layout-bar-4'),
-      {
-        onclick: () => {
-          document.querySelectorAll('.content-bar').forEach(bar => {
-            bar.style.transform = 'none'; // eslint-disable-line no-param-reassign
-          });
-          overlayVars.layout = layoutName;
-          savePrefs();
-          analytics.trackEvent('overlay', 'layout', layoutName);
-        },
+    `div.layout-btn.${layoutName}`,
+    h('div.layout-bar.layout-bar-1'),
+    h('div.layout-bar.layout-bar-2'),
+    h('div.layout-bar.layout-bar-3'),
+    h('div.layout-bar.layout-bar-4'),
+    {
+      onclick: () => {
+        document.querySelectorAll('.content-bar').forEach(bar => {
+          const newBar = bar;
```

Comment:
local variable is redundant.  just modify bar since your not reassigning and just modifying props.

Reply:
This is the only required change.

Reply:
We are dealing with `Assignment to property of function parameter 'bar'` so it won't allow reassignment of function argument's property. If that makes sense. In short
`bar.style.transform = 'none';` would give an error, what do you suggest?

Reply:
Fixed it using `setAttribute`